rocAL/rocAL_pybind/examples/rocAL_api_audio_unittest.py

```diff
@@ -0,0 +1,85 @@
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import random
+import numpy as np
+from amd.rocal.plugin.pytorch import ROCALClassificationIterator
+import torch
+# torch.set_printoptions(threshold=10_000)
+np.set_printoptions(threshold=1000, edgeitems=10000)
+from amd.rocal.pipeline import Pipeline
+import amd.rocal.fn as fn
+import amd.rocal.types as types
+import math
+# import rocal_pybind.tensor
+import sys
+import cv2
+import matplotlib.pyplot as plt
+import os
+def draw_patches(img, idx, device):
```

Comment:
I would not suggest this way of testing for audio
It is not possible from the plot what does it say, compared to image
we need to have golden outputs and compare with them, instead of plotting them as graph
we can take golden outputs from RPP itself and use them here, no need to regenerate them as well

Please change this